bes_theme/assets/data/OSSP-master.js

```diff
@@ -1477,7 +1477,302 @@ const projectOfInterestObject = {
 			]
 
 		}
-
+		{
```

Comment:
@sand-hya JSON format is not correct you need to provide comma after each object